modules/social_features/social_profile/modules/social_profile_preview/assets/js/social_profile_preview.js

```diff
@@ -41,6 +41,7 @@
                       .on('mouseleave', function () {
                         timeouts[selector] = window.setTimeout(function () {
                           dialogs[selector].close();
+                          dialogs = [];
```

Comment:
It's affected performance because sends a request each time when the modal window is opening. I'd suggest doing it at least only for the visited user modal window. But will be better to avoid it and clear this "cache" for modal windows only when the user press the "Follow" or "Unfollow" button.